README.md

```diff
@@ -1,13 +1,26 @@
-# dave
+# Dave
+
 Dave cuts VoIPGRID releases.
 
-## Installation
+## Status
+
+Maintained
+
+## Usage
+
+### Requirements
+
+You'll need a Github access token, which you can generate
+[here](https://github.com/settings/tokens). Dave only needs `repo` scope to
+work.
+
+### Installation
 
 Quite simple: get [Go](https://golang.org/dl), set a `$GOPATH`, and run
 
     go get github.com/VoIPGRID/dave
 
-## Use
+### Running
```

Comment:
I think `Use` is much more in line with `Status`, `Requirements` and `Installation`, and in my opinion these don't need to nest below `Usage`.